pkg/logs/logs.go

```diff
@@ -51,21 +48,11 @@ const (
 	TraceLevel        = 5
 )
 
-// GlogWriter serves as a bridge between the standard log package and the glog package.
-type GlogWriter struct{}
-
-// Write implements the io.Writer interface.
-func (writer GlogWriter) Write(data []byte) (n int, err error) {
-	klog.Info(string(data))
-	return len(data), nil
-}
-
 // InitLogs initializes logs the way we want for kubernetes.
 func InitLogs() {
 	logs.InitLogs()
 
-	log.SetOutput(GlogWriter{})
-	log.SetFlags(0)
+	klog.EnableContextualLogging(true) // Enable contextual logging
```

Comment:
This undoes the value set here: https://github.com/kubernetes/component-base/blob/b0a6e40497ae37a69c6396af91fb95b666b86bdc/logs/logs.go#L172